packages/client/src/Proteus.js

```diff
@@ -293,11 +314,11 @@ export default class Proteus {
   }
 }
 
-//Helper function to generate GUID-ish IDs, should a user not provide one
+// Helper function to generate GUID-ish IDs, should a user not provide one
 function uuidv4() {
-  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-    var r = (Math.random() * 16) | 0,
-      v = c == 'x' ? r : (r & 0x3) | 0x8;
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
```

Comment:
If there is a real library that does this, we should do that - this is a stackexchange snippet.

Reply:
Updated with a library we've used on the frontend.